lib/samples/apply_style_to_wms_layer/apply_style_to_wms_layer.dart

```diff
@@ -0,0 +1,164 @@
+// Copyright 2024 Esri
```

Comment:
```suggestion
//
// Copyright 2024 Esri
```
I think the copyright is missing something. It should get generated correctly, if it was incorrect in the generation we may need to revisit. Or if it was a copy-paste thing then that's fine, just correct here.

Reply:
In this case, it's incorrectly generated. Most of the samples follow the previous format. I've fixed it, particularly for this sample but we may need to revisit the generation as you said.